### PR TITLE
See message detail

### PR DESCRIPTION
**Pull request by karissa**
This also refactors the Message rendering into it's own class component which hides a lot of the logic from the chat view container, and makes it reusable for message detail.

### CODE REVIEW

**Comment by karissa** [comment]
screenshot from 2018-11-16 16-18-52

**Comment by karissa** [comment]
okay ready for review @ralphtheninja @Jikstra

File: src/renderer/components/Message.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+const tx = window.translate
+
+const GROUP_TYPES = [
+  C.DC_CHAT_TYPE_GROUP,
+  C.DC_CHAT_TYPE_VERIFIED_GROUP
+]
+
+function render (props) {
+  const { message, onClickSetupMessage } = props
+  let body = <RenderMessage {...props} />
```
[the remark this answers is not in the document]
Comment (karissa) [reply src/renderer/components/Message.js]: it's called a deconstructor https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

**Comment by ralphtheninja** [comment]
Message Detail on my own messages says "To Me":

screenshot from 2018-11-17 01-44-49

**Comment by ralphtheninja** [comment]
Maybe it should always say `From`?

**Comment by ralphtheninja** [comment]
Also has a strange JPG file icon-thingie.

**Comment by karissa** [comment]
ok fixed the issues @ralphtheninja

**Pull request merged**